src/Components/Web.Extensions/src/InputFile/RemoteBrowserFileStreamOptions.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Components.Web.Extensions
+{
+    /// <summary>
+    /// Repesents configurable options for <see cref="RemoteBrowserFileStream"/>.
+    /// </summary>
+    public class RemoteBrowserFileStreamOptions
+    {
+        /// <summary>
+        /// Gets or sets the maximum chunk size for file data sent over a SignalR circuit.
+        /// This only has an effect when using Blazor Server.
+        /// </summary>
+        public int ChunkSize { get; set; } = 20 * 1024; // SignalR limit is 32K.
```

Comment:
nit: I would use `Segment` instead of `Chunk` it's easier for non-native English speakers.